src/fillbroadcast.jl

```diff
@@ -247,3 +247,6 @@ broadcasted(::DefaultArrayStyle{N}, ::typeof(Base.literal_pow), ::Base.RefValue{
 broadcasted(::DefaultArrayStyle{N}, ::typeof(Base.literal_pow), ::Base.RefValue{typeof(^)}, r::Ones{T,N}, ::Base.RefValue{Val{k}}) where {T,N,k} = Ones{T}(axes(r))
 broadcasted(::DefaultArrayStyle{N}, ::typeof(Base.literal_pow), ::Base.RefValue{typeof(^)}, r::Zeros{T,N}, ::Base.RefValue{Val{0}}) where {T,N} = Ones{T}(axes(r))
 broadcasted(::DefaultArrayStyle{N}, ::typeof(Base.literal_pow), ::Base.RefValue{typeof(^)}, r::Zeros{T,N}, ::Base.RefValue{Val{k}}) where {T,N,k} = Zeros{T}(axes(r))
+
+# supports structured broadcast
+LinearAlgebra.fzero(x::Zeros) = zero(eltype(x))
```

Comment:
If we want to be conservative, we may define this as
```julia
# Hook into structured broadcasting in LinearAlgebra
# Since this is an undocumented internal function,
# we avoid breakages by adding this method only if `fzero` is defined at all
if isdefined(LinearAlgebra, :fzero)
    LinearAlgebra.fzero(x::Zeros) = zero(eltype(x))
end
```
This would avoid breakages if `fzero` is removed from `LinearAlgebra`. Also, missing newline at the end of this file.